src/SFA.DAS.EmployerIncentives.Web/ViewModels/Apply/SelectApprenticeships/SelectApprenticeshipsRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerIncentives.Web.ViewModels.Apply.SelectApprenticeships
+{
+    public class SelectApprenticeshipsRequest
+    {
+        public string AccountId { get; set; }
+
+        public string AccountLegalEntityId { get; set; }
+
+        public List<string> SelectedApprenticeships { get; set; }
+
+        public static string SelectedApprenticeshipsPropertyName => nameof(SelectedApprenticeships);
```

Comment:
Don't think this is need here.  It is only used by the view so should be moved to the viewModel.

Reply:
it is used by the View to bind to a property in this class. this ensures compatibility. 💁‍♂️
why move where? @Stephek